spec/models/meeting_spec.rb

```diff
@@ -4,23 +4,31 @@
   include_examples "Invitable", :meeting_invitation, :meeting
 
   context 'validations' do
-    subject { Meeting.new }
+    subject(:meeting) { Fabricate(:meeting) }
+    it {is_expected.to validate_presence_of(:date_and_time) }
+    it {is_expected.to validate_presence_of(:venue) }
+
+    context '#slug' do
+      it 'does not validate if slug present' do
```

Comment:
the wording here is a bit confusing.  Something like `fail when slug not present` would read better in this instance